PeerioTechnologies/peerio-desktop
# fix: delete received file from own Files view

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/13945/desktop-delete-option-missing-from-three-dot-menu-for-files-received-in-dms
 
#### Testing instructions  
* User should be able to delete a received file (i.e. a file they do not own) from their own Files view. This will not remove the file from the DM in which it is received.
* The deletion confirmation message only asks to confirm that user wants to remove the file. "file will be unshared..." message should not show here

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
